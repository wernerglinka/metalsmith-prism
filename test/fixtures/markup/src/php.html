<!DOCTYPE html>
<html>
  <body>
    <pre><code class="language-php">
&lt;?php
class HelloWorld {
    private string $greeting = "Hello World!";

    public function sayHello(): string {
        return $this->greeting;
    }
}
  </code></pre>
  </body>
</html>