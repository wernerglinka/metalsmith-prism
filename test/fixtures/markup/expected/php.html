<!DOCTYPE html>
<html>
  <body>
    <pre><code class="language-php"><span class="token operator">&lt;</span><span class="token operator">?</span>php
<span class="token keyword">class</span> <span class="token class-name-definition class-name">HelloWorld</span> <span class="token punctuation">{</span>
    <span class="token keyword">private</span> <span class="token keyword type-declaration">string</span> <span class="token variable">$greeting</span> <span class="token operator">=</span> <span class="token string double-quoted-string">"Hello World!"</span><span class="token punctuation">;</span>

    <span class="token keyword">public</span> <span class="token keyword">function</span> <span class="token function-definition function">sayHello</span><span class="token punctuation">(</span><span class="token punctuation">)</span><span class="token punctuation">:</span> <span class="token keyword return-type">string</span> <span class="token punctuation">{</span>
        <span class="token keyword">return</span> <span class="token variable">$this</span><span class="token operator">-&gt;</span><span class="token property">greeting</span><span class="token punctuation">;</span>
    <span class="token punctuation">}</span>
<span class="token punctuation">}</span></code></pre>
  </body>
</html>